### ISSUE TITLE
.config directory error on debian stretch

### ISSUE THREAD

**Issue by awb99** [ncw/rclone#1787]
I have installed rclone on Debian stretch, via the normal package manager.
As root user, it all works fine, and I have found the config under 
```
   ls /root/.config/rclone
   rclone.conf
```

When I run the same batch file in a crontab entry, then I get the following error:
```
   2017/10/31 10:20:02 Config file "/root/.rclone.conf" not found - using defaults
```

So it is obvious that it is looking for the config file in another place.
I solved the issue by copying the config over. However I still think it is a bug.
```
cp /root/.config/rclone/rclone.conf /root/.rclone.conf
```

> What is your rclone version (eg output from `rclone -V`)
rsync  version 3.1.2  protocol version 31

> Which OS you are using and how many bits (eg Windows 7, 64 bit)
debian stretch

> Which cloud storage system are you using? (eg Google Drive)

> The command you were trying to run (eg `rclone copy /tmp remote:tmp`)

> A log from the command with the `-vv` flag (eg output from `rclone -vv copy /tmp remote:tmp`)

**Comment by ncw** [comment]
It will look in .config first and then in .rclone.conf - it is just reporting the last place it looked.

The message could be more descriptive though, I agree.

**Comment by ncw** [comment]
Actually I think this is fixed in rclone 1.38 - try [the latest beta](https://beta.rclone.org) or [the latest release](https://rclone.org/downloads/)
